tools/src/versioning/android/unversionablePackages.json

```diff
@@ -4,6 +4,7 @@
   "expo-dev-launcher",
   "expo-dev-menu",
   "expo-dev-menu-interface",
+  "expo-eas-client-id",
```

Comment:
It's also necessary to add `expo.modules.easclientid` namespace to `android-packages-to-keep.txt`, otherwise references to this namespace in versionable packages would be versioned by the script.